### DISCUSSION TITLE
bonsai/DLC issue

### DISCUSSION THREAD

**Discussion by emilyjanedennis** [bonsai-rx/bonsai#902] [Q&A] [answered]
https://github.com/bonsai-rx/deeplabcut/issues/11#issue-1231589318

not sure if the DLC repo is monitored so posting here - would love any help/advice on how to debug or solve failed DetectPose node, thanks!

**Comment by bruno-f-cruz** [comment, marked as answer]
Try to convert your image to bgr with `ConvertColor`. This should fix it.

https://github.com/bonsai-rx/deeplabcut/issues/3
